examples/fabric/dcgan/train_torch.py

```diff
@@ -0,0 +1,271 @@
+"""
+DCGAN - Raw PyTorch Implementation
+
+Code adapted from the official PyTorch DCGAN tutorial:
```

Comment:
What exactly did you adapt from this file? Or are there no changes from the reference?

Reply:
I don't remember. Took the code, formatted it. 
This is just to say where it came from. If we ever make changes, the comment covers it.